### ISSUE TITLE
Task: Ensure accessibility

### ISSUE THREAD

**Issue by vscps** [vscps/capstone-project-activity-planner#88]
Ensure accessibility throughout the app by implementing semantic HTML where missing and adding aria and role properties where necessary.
